### ISSUE TITLE
cannot install

### ISSUE THREAD

**Issue by georgesuba** [kalvn/Shaarli-Material#64]
hi.. in readme.md is how to install this theme...

`In your Shaarli installation, open the data/config.json.php and change the raintpl_tpl parameter from tpl\/ to tpl\/material\/.`

When looking at data folder there is nothing like config.json.php..  so I've created one with your example

```
{
    "resource": {
        "raintpl_tpl": "tpl\/material\/"
    },
    
    "config": {
        "MATERIAL_PHP_DATE_PATTERN": "%d\/%m\/%Y %H:%M:%S",
        "MATERIAL_NO_QRCODE": true,
        "MATERIAL_DATE_FROMNOW": true,
        "MATERIAL_DATE_PATTERN": "DD/MM/YYYY HH:mm:ss",
        "MATERIAL_COLOR": "#607D8B",
        "MATERIAL_COLOR_FOCUS": "#546E7A",
        "MATERIAL_COLOR_ACTIVE": "#455A64"
    }
}
```

but theme is still not loading - Im using latest original repo or shaarli.

Thanks.

**Comment by kalvn** [comment]
By default, there is indeed nothing in the data folder. To create all the files, you need to first do the starting setup of your installation by just going with your browser where you installed Shaarli.
Once you defined your settings as well as your username and password, the `config.json.php` file should be present and then you'll be able to modify the `resource/raintpl_tpl` value.

Another solution, if you installed the very very last master version of both Shaarli and Shaarli Material is to go into the page `Tools > Configure your shaarli` and to change the theme with the dedicated dropdown list.

**Comment by georgesuba** [comment]
Hi, well seems to me that I get complete different options on my shaarli after update and on new install too. Are you using same original [shaarli source](https://github.com/shaarli/Shaarli)? Because even when I done new install this is what was created in my /data folder

* datastore.php
* lastupdatecheck.txt
* updates.txt
* config.php
* ipbans.php
* log.txt
* .htaccess

Also as you said I've installed latest shaarli and shaarli material theme but `Tools > Configure your shaarli and to change the theme with the dedicated dropdown list.` - There is nothing like drop down list.. main menu looks like this:

![shaarli config page](https://i.imgsafe.org/d13770c7b4.jpg)

That is all I've got there. - What I am missing?
